### PR TITLE
Fixed key names for Apple command keys

### PR DESCRIPTION
**Pull request by ffxsam**
I noticed "right click" coming up when I pressed the right command key on my Apple keyboard. I've fixed both left/right cmd key names.

### CODE REVIEW

**Comment by ffxsam** [comment]
@timoxley Thoughts?

File: index.js [hunk 1]
```diff
@@ -63,8 +63,8 @@ var codes = exports.code = exports.codes = {
   'down': 40,
   'insert': 45,
   'delete': 46,
-  'command': 91,
```
Comment (timoxley) [inline index.js]: Should probably alias left command and command, in case someone currently relying on current 'command' result.

**Comment by timoxley** [comment]
Yeah this is a weird bug, clearly a malapropism.

**Comment by ffxsam** [comment]
Updated branch to put `command` back in.

**Comment by ffxsam** [comment]
Woops, that was dumb. I trashed my fork.. but apparently a merge is still possible? let me know.

**Comment by timoxley** [comment]
Thanks for this. Released in [2.1.2](https://github.com/timoxley/keycode/compare/2.1.1...2.1.2)

**Pull request merged**